src/lib/data/parser/AccessKind.h

```diff
@@ -5,6 +5,7 @@
 
 enum AccessKind
 {	 // these values need to be the same as AccessKind in Java code
+	ACCESS_UNKNOWN = -1,
```

Comment:
I'm rather against adding this new enum constant just to return it in the `default:` case, because then we would need to handle the new value in the rest of the code base. Please remove this change and see the comments I made at the location where this enum constant would be used.

Reply:
done